setup.py

```diff
@@ -346,8 +346,7 @@ def build_extension(self, ext):
                 if hasattr(self, "parallel") and self.parallel:
                     build_args += ["-j{}".format(self.parallel)]
                 else:
-                    # Let CMake determine the parallelism to use
-                    build_args += ["-j"]
+                    build_args += ["-j1"]
```

Comment:
We can probably just remove this else statement? iirc before I made the change, we didn't add the argument at all.